### ISSUE TITLE
Breaks quotes inside of quotes

### ISSUE THREAD

**Issue by KTibow** [sveltejs/prettier-plugin-svelte#320]
If you have something like `<Component prop='Something in "quotes".' />` this plugin turns it into `"Something in "quotes"."` which is invalid Svelte.

**Comment by KTibow** [comment]
This might be relevant:
https://github.com/sveltejs/prettier-plugin-svelte/blob/ebd7e9a0dd7d5096098e16d74e6d88b9e249916f/src/print/index.ts#L433

**Comment by dummdidumm** [comment]
Related to #253

**Comment by KTibow** [comment]
Yeah I saw that but didn't fully read it, I wouldn't mind if you merged this issue into that one, but I'm fine using this one for tracking separately

**Comment by lianghai** [comment]
I’m also affected by this, but have been procrastinating about the bug report… Thanks, OP!

**Comment by lianghai** [comment]
Any progress on this?
